docs/working-copy.md

```diff
@@ -12,12 +12,16 @@ working-copy contents when they have changed. Most `jj` commands you run will
 commit the working-copy changes if they have changed. The resulting revision
 will replace the previous working-copy revision.
 
-Also unlike most other VCSs, added files are implicitly tracked. That means that
-if you add a new file to the working copy, it will be automatically committed
-once you run e.g. `jj st`. Similarly, if you remove a file from the working
-copy, it will implicitly be untracked. To untrack a file while keeping it in
-the working copy, first make sure it's [ignored](#ignored-files) and then run
-`jj file untrack <path>`.
+Also unlike most other VCSs, added files are implicitly tracked by default. That
+means that if you add a new file to the working copy, it will be automatically
+committed once you run e.g. `jj st`. Similarly, if you remove a file from the
```

Comment:
(Optional)

I think the "Similarly, if you remove a file..." sentence is a bit confusing. Perhaps we should delete it? I am tempted to say that the path is still tracked, but there is no file there. I think what happens when you delete a file depends also on whether the path is in `.gitignore`.

Reply:
I don't find it confusing, and I don't think I changed that part anyway, so I'll leave it as is.